src/contracts/AccountingEngine.sol

```diff
@@ -193,7 +193,7 @@ contract AccountingEngine is Authorizable, Modifiable, Disableable, IAccountingE
 
   /// @inheritdoc IAccountingEngine
   function auctionSurplus() external returns (uint256 _id) {
-    if (_params.surplusIsTransferred == 1) revert AccEng_SurplusAuctionDisabled();
+    if (_params.surplusIsTransferred == 100) revert AccEng_SurplusAuctionDisabled();
```

Comment:
this should be in `WAD %` units

Reply:
isn't `_params.surplusIsTransferred` in RAD?